### PR TITLE
Bundle for Openshift

### PR DESCRIPTION
**Pull request by spkesan**
- Add labels for RedHat Openshift platform
- Don't specify securityContext in deployment spec in CSV
- Add more lightweight CR configuration sample - `dim_nostorage_cluster_cr_skip_validation.yaml`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -187,6 +191,23 @@ bundle: manifests kustomize
 		sed -i "s@containerImage: controller:latest@containerImage: $(IMG)@g" bundle/manifests/aerospike-kubernetes-operator.clusterserviceversion.yaml; \
 	fi
 
+# OPENSHIFT_VERSION has to be defined to build bundle for openshift platform
```
Comment (ashishshinde) [inline Makefile:194]: Can the same bundle with these extra label work on regular kubernetes. If so we can always add openshift labels.

File: config/manifests/bases/aerospike-kubernetes-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -35,8 +35,8 @@ spec:
...
...A+NCaoYObIYMuQJAV7/Mq618ZcR+d5yRfX7g119qwXimJNVyAJBLjo9CD7eL+pwMzEoHVGtgX/c47xW5Vejtf5FXXfGDolRLqoh8CQCUyPPFKRnn8movHhhNwcXm9BE/U9Wv/2o8xuq8yS8CS1fqeQxec4ER/zfW0Pd3OoI2A4BosqU0NeN+V2Z1DykhCxqyRmhTBYB7/p/WMdUg40fOy1+UyPOl5plnc6xVI17KsaqhoaM4MGoTgrR/gshTACCCJYAsUYNHxzcPk5VS/QJ8J89a8VNX2mbXpRKC3zhnnItgiYIsGXyBaBhyiNmv6Xj6ngamJ6i8moovl6RaZjgnscKk3ihOszyYV11xzOVCNG3EMUZTgNoDx1lcn+XzE0dLU1MzoTnPDCqbvVd+MtJ7cmpqLkCpwQFlHfrIF6pCRhX+Aqlz8ZWD5WoaTHd6RV7Nxe8AanBKgC6mt0qnZ05xpfS8q1WXA/u7vyS62gql9qo7nN1UCtXQ1S9F1/PzalwLKwDIt1aU59dcfE6UvhJK/yGgztyh7BtQ6mXR9XxX7o7g6DvP3GVkaP/era+He8/dvgMC2EN6Ox9XyjmJVqKh8Jqrn9Eo9HiiXyYSn9/DCrHZLndqWoZmMtlNpn573rVql8ZIgtvUE70RSAAAk8k+4mC2ZpcnlOghI5fsIEpz6ct1a7m25ru9Lqyn4wvdgRHfBgBdt4cnhWguj6MN7NX85Whc+gE9DJH2AEmww55pUmhXSrtgl94uWK03hrvX03ByayqLFFBcOjn9FwhC5NDnAnXYEkLUDUtllc3V8kx22xGlQmeOpS3Dstt+Lgh+eaS+vtt7hvtMs65ebSw0m5cH6EFhjtfqoUnBatjPSOxqVNto0m0+f/EzEREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREREZHx/X/7J9jNSPekbAAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAyMC0wNy0xNFQwNjowMToyNyswMDowMN2Q520AAAAldEVYdGRhdGU6bW9kaWZ5ADIwMjAtMDctMTRUMDY6MDE6MjcrMDA6MDCszV/RAAAAAElFTkSuQmCC
```
Comment (ashishshinde) [inline config/manifests/bases/aerospike-kubernetes-operator.clusterserviceversion.yaml:38]: The original is an optimized webp scalable graphics image which renders well all scales. Unless this png is a different logo we can keep the webp image

Comment (spkesan) [reply config/manifests/bases/aerospike-kubernetes-operator.clusterserviceversion.yaml:38]: `webp` format is not supported. Atleast not during openshift certification.

File: Makefile (hunk 1 above)
Comment (spkesan) [reply Makefile:194]: It should work for non-openshift. I'll see and make the changes.

File: config/manifests/bases/aerospike-kubernetes-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (ashishshinde) [reply config/manifests/bases/aerospike-kubernetes-operator.clusterserviceversion.yaml:38]: Can you try this. svg+xml is supported I think.

  - base64data: PHN2ZyB3aWR0aD0iMjUwMCIgaGVpZ2h0PSIyNTAwIiB2aWV3Qm94PSIwIDAgMjU2IDI1NiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiBwcmVzZXJ2ZUFzcGVjdFJhdGlvPSJ4TWlkWU1pZCI+PHBhdGggZD0iTTAgMGgyNTZ2MjU2SDBWMHoiIGZpbGw9IiNCMDI1MkEiLz48cGF0aCBkPSJNMTU2LjI5MyA5NS42MDVsLTczLjE2OSAzMi41OTQgNzMuMTcgMzIuODFWOTUuNjA1em0tOTIuMDMyIDM5Ljk3OWwtMTcuMDQ5LTcuMyAxNy4wNDktNy44ODIgMTQ0LjUyNy02NS4zNzZ2MTcuNDAzbC0zNy45MzIgMTYuODI0djc4LjExN2wzNy45MzIgMTd2MTYuNjA0TDY0LjI2IDEzNS41ODR6IiBmaWxsPSIjRkZGIi8+PC9zdmc+
    mediatype: image/svg+xml

Comment (spkesan) [reply config/manifests/bases/aerospike-kubernetes-operator.clusterserviceversion.yaml]: `svg+xml` works.

File: Makefile (hunk 1 above)
Comment (spkesan) [reply Makefile]: Added Openshift labels to common image.

**Review by ashishshinde — APPROVED** [review]

**Pull request merged**